cert/cert.go

```diff
@@ -45,16 +45,15 @@ const (
 	duration365d = time.Hour * 24 * 365
 )
 
-var (
-	ErrStaticCert = errors.New("cannot renew static certificate")
-)
+var ErrStaticCert = errors.New("cannot renew static certificate")
 
-// Config contains the basic fields required for creating a certificate
+// Config contains the basic fields required for creating a certificate.
 type Config struct {
-	CommonName   string
-	Organization []string
-	AltNames     AltNames
-	Usages       []x509.ExtKeyUsage
+	CommonName       string
+	Organization     []string
+	AltNames         AltNames
+	Usages           []x509.ExtKeyUsage
+	ExpirationInDays time.Duration
```

Comment:
I think `ExpirationInDays` might not be the correct name for this variable as it is a `time.Duration` instead of an int or something.